src/Tools/GenerateDocumentationAndConfigFilesForBrokenRuntime/GenerateDocumentationAndConfigFilesForBrokenRuntime.csproj

```diff
@@ -13,7 +13,6 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.CodeAnalysis" Version="$(MicrosoftCodeAnalysisExecutableVersion)" />
-    <PackageReference Include="System.Collections.Immutable" Version="$(SystemCollectionsImmutableExecutableVersion)" />
-    <PackageReference Include="System.Reflection.Metadata" Version="$(SystemReflectionMetadataExecutableVersion)" />
+    <PackageReference Include="System.Composition" Version="8.0.0-preview.4.23220.1" />
```

Comment:
Temporary change to make the build consume source-build-reference-package dependency describe in https://github.com/dotnet/roslyn-analyzers/issues/6606#issuecomment-1520341720

> The package is loaded during the build and an SBRP for it exists. When a dependency on SBRP is added, the ref assembly for the package is picked up, crashing the build. I circumvented this by forcing the latest version of the package, which might not be desirable